overlay/usr/lib/inithooks/firstboot.d/20regen-mayan-secrets

```diff
@@ -3,30 +3,41 @@
 
 . /etc/default/inithooks
 
-SV_CONF=/etc/supervisor/conf.d/mayan.conf
+MAYAN_CONF=/opt/mayan-edms/media/config.yml
 RE_CONF=/etc/redis/redis.conf
+RMQ_CONF=/etc/rabbitmq/rabbitmq-env.conf
 ENV=/etc/mayan/env
 DB_USER=mayan
 
 # Create new passwords
 DB_PASS=$(mcookie)
 RE_PASS=$(mcookie)
+RMQ_PASS=$(mcookie)
 
 # Update DB password in Supervisor config, as well as env file
-sed -i "\|MAYAN_DATABASES|s|D: [0-9a-z]*,|D: $DB_PASS,|" $SV_CONF
+sed -iE '/^DATABASES/,/^[^ \t]/ s/PASSWORD:.*/PASSWORD: '$DB_PASS'/g' $MAYAN_CONF
 sed -i "\|PASSWORD|s|:.*|:'$DB_PASS',|" $ENV
 
 # Update the DB user password in postgres
 $INITHOOKS_PATH/bin/pgsqlconf.py --user=$DB_USER --pass="$DB_PASS"
 
 # regen the Django secret
-PYTHONPATH=/opt/mayan-edms/lib/python3.7/site-packages $INITHOOKS_PATH/bin/mayan-gen-secret.py
+#. /opt/mayan-edms/bin/activate
+#python $INITHOOKS_PATH/bin/mayan-gen-secret.py
+#deactivate
```

Comment:
Is the Django secret(s) regenerated some other way? Or did you just forget to uncomment it?

Regardless, if it's no longer needed, we should just remove it IMO.